### ISSUE TITLE
sys.flags.utf8_mode not supported

### ISSUE THREAD

**Issue by dumblob** [google/pytype#389]
```
import sys
assert( sys.flags.utf8_mode == True )
```
```
Computing dependencies
Analyzing 5 sources with 1 local dependencies
ninja: Entering directory `/home/user/app/.pytype'
[1/2] check test00
FAILED: /home/user/app/.pytype/pyi/test00.pyi 
pytype-single --imports_info /home/user/app/.pytype/imports/test00.imports --module-name test00 -V 3.7 -o /home/user/app/.pytype/pyi/test00.pyi --analyze-annotated --nofail --quick /home/user/app/test00.py
File "/home/user/app/test00.py", line 2, in <module>: No attribute 'utf8_mode' on sys._flags [attribute-error]

For more details, see https://google.github.io/pytype/errors.html#attribute-error.
[2/2] check main
ninja: build stopped: subcommand failed.
```

**Comment by rchen152** [comment]
Looks like another Python 3.7 addition (https://docs.python.org/3/library/sys.html#sys.flags) that needs to be added to tyepshed: https://github.com/python/typeshed/blob/b873894e169b0b15742eb8931f5320347b787dcc/stdlib/3/sys.pyi#L69

**Comment by rchen152** [comment]
https://github.com/python/typeshed/pull/3205

**Comment by dumblob** [comment]
Feel free to close this after the new `pytype` (with the corresponding `typeshed`) gets released.

**Comment by rchen152** [comment]
Sorry for the long wait. This fix is now in pytype 2019.09.17.
